tests/ImageSharp.Tests/ImageSharp.Tests.csproj

```diff
@@ -23,6 +23,7 @@
   </Choose>
   
   <ItemGroup>
+    <DotNetCliToolReference Include="dotnet-xunit" />
```

Comment:
We shouldn't need this. It's in the shared props/targets

Reply:
removed